src/components/screens/error-page.tsx

```diff
@@ -0,0 +1,25 @@
+import { type ReactElement } from "react";
+import { useRouteError } from "react-router-dom";
+
+/**
+ * Error page for displaying error status text and message
+ * 
+ * @returns ReactElement
+ */
+export default function ErrorPage (): ReactElement {
```

Comment:
Export defaults as the last line of the file

Reply:
added export to last line